frontend/appflowy_flutter/assets/translations/hi.json

```diff
@@ -0,0 +1,441 @@
+{
+  "appName": "ऐप फ्लोवी",
+  "defaultUsername": "में",
+  "welcomeText": "स्वागत है @:appName",
+  "githubStarText": "गिटहब पैर स्टार दे",
+  "subscribeNewsletterText": "हमारे न्यूज़ की जानकारी पाए",
+  "letsGoButtonText": "शुरू करे",
+  "title": "टाइटल",
+  "signUp": {
+    "buttonText": "साईन करे",
+    "title": "साईन करे @:appName",
+    "getStartedText": "शुरू करे",
+    "emptyPasswordError": "पासवर्ड खाली नहीं रख सकते",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है",
+    "alreadyHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "repeatPasswordHint": "रिपीट पासवर्ड"
+  },
+  "signIn": {
+    "loginTitle": "लॉगिन करे @:appName",
+    "loginButtonText": "लॉगिन",
+    "loginAsGuestButtonText": "शुरू करे",
+    "buttonText": "साईन करे",
+    "forgotPassword": "क्या आप पासवर्ड भूल गए ?",
+    "emailHint": "ईमेल",
+    "passwordHint": "पासवर्ड",
+    "dontHaveAnAccount": "क्या आपके पास पहले से ही खाता है ?",
+    "repeatPasswordEmptyError": "रिपीट पासवर्ड खाली नहीं रख सकते",
+    "unmatchedPasswordError": "रिपीट पासवर्ड और पहला पासवर्ड सामान नहीं है"
+  },
+  "workspace": {
+    "create": "काम करने की जगह बनाये",
+    "hint": "काम करने की जगह",
+    "notFoundError": "काम करने की जगह नहीं मिली"
+  },
+  "shareAction": {
+    "buttonText": "शेयर करे",
+    "workInProgress": "जल्द आ रहा है",
+    "markdown": "मार्क कीजिये",
+    "copyLink": "लिंक कॉपी करे"
+  },
+  "moreAction": {
+    "small": "छोटा",
+    "medium": "बड़ा",
+    "large": "बहोत बड़ा",
+    "fontSize": "अक्षर साइज",
+    "import": "बहार से लाये",
+    "moreOptions": "और विकल्प"
+  },
+  "disclosureAction": {
+    "rename": "नाम बदले",
+    "delete": "इसे मिटायें",
+    "duplicate": "डुप्लिकेट"
+  },
+  "blankPageTitle": "खली पेज",
+  "newPageText": "नया पेज",
+  "trash": {
+    "text": "कूड़ा",
+    "restoreAll": "सभी बहाल करो",
```

Comment:
`सभी पुनर्स्थापित करो` instead of `सभी बहाल करो` because we use पुनर्स्थापित in next, it is more appropriate for consistency, also बहाल can be interpreted differently